src/content/docs/codestream/start-here/codestream-new-relic.mdx

```diff
@@ -5,6 +5,8 @@ metaDescription: "Read about the useful ways CodeStream and New Relic work toget
 
 CodeStream and New Relic One work together to give you insight into your code's errors, as well as making it easier to get started instrumenting your code with our APM agents.
 
+If you haven't already, [sign up for a free New Relic account](https://newrelic.com/signup) so that you can get the most out of New Relic CodeStream.
```

Comment:
I think you can take this one out. You have a CTA lower on the page and it makes sense there.

Reply:
I thought it was important to have one as high up on the page as possible. Thanks for clarifying!